local-cli/cliEntry.js

```diff
@@ -137,7 +137,7 @@ function run() {
 
   commander.parse(process.argv);
 
-  const isValidCommand = commands.find(cmd => cmd.name === process.argv[2]);
+  const isValidCommand = commands.find(cmd => cmd.name.split(" ")[0] === process.argv[2]);
```

Comment:
Yeah, but we have already merged a fix for that here: https://github.com/facebook/react-native/blob/master/local-cli/cliEntry.js#L161